pages/linux/eyewitness.md

```diff
@@ -0,0 +1,24 @@
+# EyeWitness
+
+> Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
+> More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.
+
+- Scan a single website using `--single` flag:
+
+`EyeWitness --single {{website}}`
+
+- Scan multiple urls from file and output to foldername:
+
+`eyewitness -f {{filename}} -d {{foldername}}`
```

Comment:
```suggestion
`eyewitness -f {{path/to/file}} -d {{path/to/directory}}`
```

Reply:
Note for the PR author: it's a path placeholder format [violation](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): mandatory fix.